src/components/FilterButtons.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Button from 'components/ui/Button';
+
+function FilterButtons() {
+  const filterButtons = [{ label: 'All' }, { label: 'Incomplete' }, { label: 'Complete' }];
```

Comment:
import {LABEL_ALL, LABEL_INCOMPLETE, LABEL_COMPLETE} from 'consts'
```suggestion
  const filterButtons = [{ label: LABEL_ALL }, { label: LABEL_INCOMPLETE }, { label: LABEL_COMPLETE }];
```